en/docs/chapter_graph/graph.md

```diff
@@ -19,18 +19,18 @@ If vertices are viewed as nodes and edges as references (pointers) connecting th
 Based on whether edges have direction, graphs can be divided into <u>undirected graphs</u> and <u>directed graphs</u>, as shown in the figure below.
```

Comment:
To express it more coherently, this sentence could be updated to"Graphs can be classified into two main categories based on the presence or absence of directionality in their edges: undirected graphs and directed graphs. The figure provided below shows this distinction. "